External/SPMmodified/xASL/xASL_io_SaveNifti.m

```diff
@@ -16,6 +16,9 @@ function xASL_io_SaveNifti(pathOrigNifti, pathNewNifti, imNew, nBits, bGZip, cha
 %                  masks, since it cannot contain a large data range
 %   bGZip          Gzip the result (OPTIONAL, DEFAULT 1)
 %   changeMat      New orientation matrix 4x4 (OPTIONAL, DEFAULT same as previous)
+%   bSaveJson      Also save a JSON sidecar file with the NIfTI (OPTIONAL, DEFAULT = false)

```

Comment:
This activates the saving, but also activates the loading from the reference JSON.

But what if we want to add new JsonFields, but not use the reference (cause it's completely off).
Sort of bSaveJson = false but JsonFields not empty.

Reply:
bSaveJson isn't about saving JSON sidecar, but about taking the JSON from pathOrigNifti.